Updated Screenshots (#3182)
spectrocloud/librarium: docs/docs-content/user-management/saml-sso/palette-sso-with-keycloak.md

    | ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
    | **Issuer URL**    | Your Keycloak URL with `/realms/master` appended to the end. For example, `http://keycloak.dmitry.sa.spectrodemos.com:8080/realms/master`. |
    | **Client ID**     | The client ID you created in the previous steps. In this guide, the name `palette` was used.                                               |
    | **Client Secret** | The client secret you retrieved in the previous step.                                                                                      |

    ![View of the Palette SSO screen](/keycloak/user-management_saml-sso_keycloak-08-palette-callback.webp "Palette Callback")

15. Scroll down to the bottom of the page, and click on **Enable**. If all values are correct, you will receive green
    message stating "OIDC configured successfully" at the top right corner.

16. Navigate back to the Keycloak admin console. In the left **Main Menu**, select **Users**. Fill in first name, second
    name, and the email address of the admin user.

>     ![alt_text](/keycloak/user-management_saml-sso_keycloak-10-keycloak-admin.webp "Keycloak Admin")

Reviewer: Should we write something in `alt_text` perhaps?

>     ![alt_text](/keycloak/user-management_saml-sso_keycloak-10-keycloak-admin.webp "Keycloak Admin")

Author: ```suggestion
    ![View of the Keycloack admin portal screen](/keycloak/user-management_saml-sso_keycloak-10-keycloak-admin.webp "Keycloak Admin")
```

>     ![alt_text](/keycloak/user-management_saml-sso_keycloak-10-keycloak-admin.webp "Keycloak Admin")

Author: ```suggestion
    ![View of the Keycloack admin portal screen](/keycloak/user-management_saml-sso_keycloak-10-keycloak-admin.webp "Keycloak Admin")
```

1. Log in to Palette as a Tenant Admin. Navigate to the left **Main Menu** and select **Users & Teams**. Click on the
   **Teams** tab to access the Teams page. Click on the **Create Team** button to create a new team.

   ![Image of the Palette Tenant menu with Users & Teams highlighted](/keycloak/user-management_saml-sso_keycloak-14-palette-groups.webp "Palette groups")

2. Provide a team name, such as `admins` but leave the **Members** field empty. Click on **Confirm** to create the team.

3. Next, select the **admins** row to access the team settings page. Click on **New Project Role** and add the **Project
   Admin** role to the **Default** project. Click on **Confirm** to save the changes.

>    ![Image of project roles highlighteds](/keycloak/user-management_saml-sso_keycloak-15-palette-project.webp "Palette Project")

Reviewer: ```suggestion
   ![Image of project roles highlighted](/keycloak/user-management_saml-sso_keycloak-15-palette-project.webp "Palette Project")
```

4. From the left **Main Menu**, select **Tenant Settings**. Next, click on **SSO** to access the SSO configuration page.
   Click on the **OIDC** tab to access the OIDC configuration page. Scroll down to the **SSO Admins** section and select
   the **admins** team from the **Default Team** drop-down Menu. Click on **Save**.

5. Navigate back to the Keycloak admin console. From the left **Main Menu** click on **Users**. Create a user and ensure
   the field **Username**, **Email**, and **First Name** are filled out. Click on **Save** to save the user.

>    ![Keycloack's management portal with the User view in focus](/keycloak/user-management_saml-sso_keycloak-17-keycloak-user.webp "Keycloak user")

Reviewer: ```suggestion
   ![Keycloak's management portal with the User view in focus](/keycloak/user-management_saml-sso_keycloak-17-keycloak-user.webp "Keycloak user")
```

6. Next, click on the **Credentials** tab and assign the user password.

>    ![Keycloack's management portal with the User view in focus that lists all current users](/keycloak/user-management_saml-sso_keycloak-18-keycloak-password.webp "Keycloak password")

Reviewer: ```suggestion
   ![Keycloak's management portal with the User view in focus that lists all current users](/keycloak/user-management_saml-sso_keycloak-18-keycloak-password.webp "Keycloak password")
```
